Foundation/FileHandle.swift

```diff
@@ -377,6 +377,11 @@ open class FileHandle : NSObject, NSSecureCoding {
       _closeOnDealloc = closeopt
     }
 
+    public init(_handle: HANDLE, _closeOnDealloc: Bool) {
```

Comment:
Should this not be `internal` and the test wrapped in `NS_FOUNDATION_ALLOWS_TESTABLE_IMPORT` ? Otherwise its adding new API.

Reply:
This patch will be hanging for a bit as I resolve this. (I am okay having experiemental (underscored) API for the use of platforms where there is no alternative. On Windows, where `fd`s are `HANDLE`s, there's little choice.)

A nit: the underscore is only on the first parameter.

Reply:
Since I put this up, a `FileHandle(handle:closeOnDealloc:)` has been added as `internal`.  This can go away.